cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,40 @@
-describe('', () => {
-  before(() => {
+describe('Create and delete article', () => {
+  let user;
+  let article;
 
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
   });
 
-  it('', () => {
-
+  it('should create an article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.visit('/editor');
+    cy.findByyPlaceholder('Article Title').type(article.title);
```

Comment:
make sure that your cy function is written correctly) findPlaceholder or findByPlaceholder
not find**Byy**Placeholder
fix it everywhere